src/test/java/com/google/firebase/FirebaseAppTest.java

```diff
@@ -428,6 +526,22 @@ public void testFirebaseExceptionNullDetail() {
   public void testFirebaseExceptionEmptyDetail() {
     new FirebaseException("");
   }
+  
+
+  private void setDefaultConfigEnv(String configJSON) {
```

Comment:
We don't shorten names in Java. Recommend calling this: setEnvironmentVariable() or even setFirebaseConfigEnvironmentVaraible()